templates/partials/character-header.html

```diff
@@ -30,30 +30,26 @@
             {{/each}}
         </div>
     </div>
-    <div class="flexrow">
-        <div class="char-flex">
-            <label><b>Race</b></label>
-            {{race.name}}
-        </div>
-        <div class="char-flex">
-            <label><b>Gender</b></label>
-            <input name="data.gender" type="text" value="{{data.gender}}" placeholder=""/>
-        </div>
-        <div class="char-flex">
-            <label><b>Age</b></label>
-            <input class="header-stat" name="data.general.age" type="number" value="{{data.general.age}}" placeholder="0" data-dtype="Number"/>
-        </div>
-    </div>
-    <div class="flexrow">
-        <div class="char-flex">
-            <label><b>Profession</b></label>
-            {{profession.name}}
-        </div>
-        <div class="char-flex">
-            <label><b>Origin</b></label>
-            {{data.general.homeland.value}}
-        </div>
-    </div>
+    <table class="char-bottom-section">
+        <tbody>
+            <tr>
+                <td><b>Race</b></td>
```

Comment:
Can use this opportunity to change it and localized those labels

Reply:
Sure! Added in 3960fa1eb4002dbd24ead66153ce8d321abc1cfd